kubernetes/manifest-templates/pharos.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   hostNetwork: true
   containers:
   - name: pharos-dashboard
-    image: ereslibre/pharos:development
+    image: mmeister/pharos_dev:latest
```

Comment:
agree, this should be only temporary until we have some repo. maybe we could use the opensuse namespace